app/queries/find_many_by_non_string_property.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class FindManyByNonStringProperty
```

Comment:
Do we need both this class and `FindManyByStringProperty`? We worked around a similar difference to create a single `FindByProperty` query: https://github.com/pulibrary/figgy/pull/2472/files#diff-795a87e9dbaac53a47575a81d2aa06f2

Reply:
I hadn't noticed that. I'll see if I can reconfigure the query classes.

Reply:
@escowles Created a new FindManyByProperty query class to replace those two.